test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/../common/pywrap", sys.argv)
+
+from testlib import MachineCase, test_main
+
+
+class TestWireGuard(MachineCase):
```

Comment:
Please add "WireGuard" to `RE_NETWORKING` in test/run, so that it runs in the "/network" scenario instead of "/other".